site/content/docs/user-guide/search.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Discovering Plugins
+slug: discovering-plugins
+weight: 300
+---
+
+You can find a list of `kubectl` plugins distributed on Krew [here][list].
+
+On your machine, there's a local copy of which plugins are available on Krew. To
+update your local copy of the plugin list, run:
+
+## List all plugins
```

Comment:
I would avoid talking about `list plugins` because it can be so easily mixed up with `krew list`.

Maybe combine this point and the next one under one common heading `Search plugins`?